zulipterminal/cli/run.py

```diff
@@ -87,7 +87,18 @@ def fetch_zuliprc(zuliprc_path: str) -> None:
                       "\nPlease enter your credentials to login into your"
                       " Zulip organization."
                       "\n" +
-                      in_color('blue', "Zulip-Realm URL: "))
+                      "\nNOTE: The " + in_color('blue', "Zulip URL") +
+                      " is where you would go in a "
+                      "web browser to log in to Zulip." +
+                      "\nIt often looks like one of the following:" +
+                      in_color('green', "\n   your-org.zulipchat.com") +
+                      " (Zulip-hosted servers)" +
+                      in_color('green',"\n   zulip.your-org.com") +
```

Comment:
missing whitespace after ',' here (and another below) would fail PEP8 check.

I like the clarification added. That part was slightly confusing to me when I launched zulip-terminal for the first time.

Reply:
@mregdos Thanks for the check - I forgot to run manually, though the CI picked it up :)

Glad you liked the clarification :+1: